src/localization/ru/general.json

```diff
@@ -0,0 +1,31 @@
+{
+  "general.prev": "Предыдущее",
+  "general.next": "Далее",
```

Comment:
Is this a better translation for "Next"?

`Следующий`

Reply:
I made changes for the second and third lines because the previous translation was not correct.

```json
  "general.prev": "Предыдущее задание",
  "general.next": "Следующее задание",
```